src/foam/box/HTTPBox.js

```diff
@@ -26,8 +26,9 @@ foam.CLASS({
   ],
 
   imports: [
+    'generator? as ctxGenerator',
```

Comment:
I would have to subclass all <transport>Box/Service classes to do that. I would also risk having my code break if implementation details of the base class change underneath me. All I want to do is customize serialization, not the entire method that happens to invoke it (which right now includes special handling of RegisterSelfMessage, etc.).

I am fine with switching to `JSONParserStringifier` instead, but I assumed that others might now be. There are definitely trade-offs to making such a change. E.g., JSON is more verbose (so if bandwidth is at a premium, maybe FON is better); also, I don't know whether FON or JSON may have different relative performance for large vs. small payloads.

How do you feel about switching to `JSONParserStringifier`-equivalent logic @adamvy and @kgrgreer?